src/ConnectedMode/Migration/RoslynProjectProvider.cs

```diff
@@ -42,6 +42,7 @@ internal class RoslynProjectProvider : IRoslynProjectProvider
 
         [ImportingConstructor]
         public RoslynProjectProvider(VisualStudioWorkspace vsWorkspace)
+            : this(new VisualStudioWorkspaceWrapper(vsWorkspace))
```

Comment:
Oopsie - minor problem with the wrapper that I found when debugging...

Reply:
Ah, yea also missed that